docs-2.0/7.data-security/2.backup-restore/1.what-is-br.md

```diff
@@ -0,0 +1,69 @@
+
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
+2. The BR sends a request to create backup files.
+3. The metad process is locked.
+4. The metad process blocks writes to the specified graph spaces on the storaged process.
+5. The metad process sends a request to storaged processes for the snapthot file names.
+6. The metad process scans local RocksDB files and outputs SST files.
+7. The metad process cancels blocking writes to the graph spaces on the storaged process.
+8. The metad process sends response to the BR with the metad data including the thrift format, partition information of the graph spaces, and the log commit ID of each partition, and the snapshot information including the catalog of the snapshots of each storaged process, their SST file names of the meta server, and the backup file names.
+9. The metad process is unlocked.
+10. The BR SSH to the meta server where the leader locates and all the storage servers and backs up files. If Amazon S3 or Alibaba Cloud OSS is used, the backfiles are uploaded to the remote store.
+11. If Amazon S3 or Alibaba Cloud OSS is used, the BR calls commands to upload files to the remote store.
+12. The BR sends a request to clean the snapshots in meta servers and storage servers.
+  
+This figure shows how the backup is implemented.
+
+![The figure shows the backup procedure](../../figs/ng-ug-001.png "Implementation of backup")
+
+When backup files are generated, the file names are generated automatically. A file name is in the format of `BACKUP_YY_MM_DD_HH_mm_SS`, of which,
+
+- `BACKUP`: Indicats the files are bakup files.
+- `YY_MM_DD_HH_mm_SS`: Indicates the timestamp when a file is generated.
+
+### Restore
+
+The restoration process is as follows:
+
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
+2. The BR downloads the meta data of the backup files from the remote store or other external store.
+3. The BR verifies the topology of the cluster.
+4. The BR stops the Meta Service and the Storage Service remotely.
+5. The BR SSH logs in to meta and storage servers to remove the existing data files.
+6. When data files are removed, the BR SSH logs in to meta and storage servers and downloads the backup files from the remote store or other external store.
+7. When the backup files are downloaded, the BR starts the Meta Service.
+8. The BR calls the `br restore` command to change the partition information of the specified metad processes.
+9. The BR starts the Storage Service.
+
+This figure shows how the restoration is implemented.
+
+![The figures shows the restoration process](../../figs/ng-ug-002.png "Implementation of restoration")
+
+## Features
```

Comment:
I think features should come first instead of implementation because the reader care more about what br can do